e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -393,31 +406,44 @@ describe('Plugin Marketplace', () => {
         });
 
         it('not display any plugins and no error bar', () => {
-            // * no plugins should be visible
+            // * Verify no plugins should be visible
             cy.get('#marketplaceTabs-pane-allPlugins').findByText('There are no plugins available at this time.');
```

Comment:
This case may still fail on cypress as I believe the server has prepackaged plugins loaded. So even if remote marketplace is disabled, prepackaged plugins appear